src/main/java/com/spotify/github/v3/clients/ChecksClient.java

```diff
@@ -116,7 +132,9 @@
    */
   public CompletableFuture<CheckSuite> getCheckSuite(final String id) {
     final String path = String.format(GET_CHECK_SUITE_URI, owner, repo, id);
-    return github.request(path, CheckSuite.class, extraHeaders);
+    CompletableFuture<CheckSuite> future = github.request(path, CheckSuite.class, extraHeaders);
+    tracer.span("Get check suite", future);
+    return future;
```

Comment:
Perhaps these sites could use a helper method that accepts a `CF`/`CS`, invokes `tracer.span` but returns the input? Then these methods could go largely untouched with just 
```java
return tracer.span("Get check suite", github.request(path, CheckSuite.class, extraHeaders));
```
WDYT? Perhaps it would even magically satisfy the grumpy Codecov checks ;)